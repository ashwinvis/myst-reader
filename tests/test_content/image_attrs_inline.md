---
title: Image with attrs_inline syntax
author: My Author
Date: 2020-10-16
---
This is file contains a image.

![fishy](img/fun-fish.png){.bg-warning w=100px align=center}

![]({static}/images/foobar.png){width=30}
