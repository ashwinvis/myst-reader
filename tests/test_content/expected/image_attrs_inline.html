<p>
 This is file contains a image.
</p>



<p>
 <a class="bg-warning reference internal" href="img/fun-fish.png">
  <img alt="fishy" class="bg-warning align-center" src="img/fun-fish.png" style="width: 100px;"/>
 </a>
</p>



<p>
 <a class="reference internal" href="{static}/images/foobar.png">
  <img alt="" src="{static}/images/foobar.png" style="width: 30px;"/>
 </a>
</p>